src/ducks/categorization/services-fixtures.spec.js

```diff
@@ -0,0 +1,123 @@
+import { localModel } from "./services";
+import { tokenizer } from ".";
+import { Transaction } from "../../models";
+const path = require("path");
+
+const fixturePath = path.join(__dirname, "fixtures");
+const cat2name = require("/Users/fweber/Code/auto-bank-categorization/data/tree.json");
```

Comment:
This should be more generic. Tests in CI can't find this file to it fails. According to the name of the file, it already exists in our code: https://github.com/cozy/cozy-banks/blob/master/src/ducks/categories/tree.json. Is it the same file?

Reply:
Yes it is the same file and my mistake for the hard-link